index.js.flow

```diff
@@ -0,0 +1,574 @@
+// @flow
```

Comment:
is this file completely generated from annotations?

Reply:
no, had to write most of it, only the Token and Member class methods can be generated, all the class types are written